src/ifcconvert/IfcConvert.cpp

```diff
@@ -624,8 +626,15 @@ int main(int argc, char** argv) {
 	);
 
     path_t output_temp_filename = output_filename + IfcUtil::path::from_utf8(TEMP_FILE_EXTENSION);
+	
+	std::vector<std::wstring> tokens;
```

Comment:
You cannot just use wstring, but use path_t which correctly toggles between string and wstring depending on platform and settings https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.6.0/src/ifcconvert/IfcConvert.cpp#L69